crypto/src/node.rs

```diff
@@ -37,7 +37,22 @@ pub trait Index {
 /// Node count. Right now it doubles as node weight in many places in the code, in the future we
 /// might need a new type for that.
 #[derive(
-    Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Add, Sub, AddAssign, SubAssign, Sum, From, Into,
+    Copy,
+    Clone,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Hash,
+    Debug,
+    Default,
+    Add,
+    AddAssign,
+    From,
+    Into,
+    Sub,
+    SubAssign,
+    Sum,
 )]
```

Comment:
I agree, but this would require changing https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#merge_derives in our rustfmt.toml... I'm not sure whether I should modify our style on demand :|